.travis.yml

```diff
@@ -11,6 +11,7 @@ script:
 
 notifications:
   email: false
+  webhooks: https://www.travisbuddy.com/
```

Comment:
@manankalra Looks good. :+1: 
I also agree with you that adding "on_success: never" should be there as well. Gitmate automatically approves the PR if all CI passes, so there should ideally be no message in that case. :smile: